lms/djangoapps/teams/views.py

```diff
@@ -81,7 +74,38 @@ def team_post_save_callback(sender, instance, **kwargs):  # pylint: disable=unus
                 )
 
 
-class TeamsDashboardView(View):
+class TeamAPIPagination(pagination.PageNumberPagination):
```

Comment:
One thing I discovered while working on this PR is that we're very inconsistent in how we paginate across our APIs.  I chose to make the "default" the one that most apps seemed to be using.  If we're using the scheme that teams is using moving forward, then I can certainly make that the default and switch the other APIs to use their own custom schemes.